ansible/roles/jenkins-master/meta/main.yaml

```diff
@@ -1,3 +1,3 @@
 ---
 dependencies:
-  - role: handlers-jenkins-master
+  - role: nginx
```

Comment:
I believe this is not needed. `restart nginx` is not called in `jenkins-master` role